AchieveIt.BusinessLogic/Services/AuthService.cs

```diff
@@ -73,14 +144,41 @@ private async Task<AuthUserResultDto> RegisterUser(User user)
                     .WithPayloadData("Patronymic", personBase.Patronymic);
             }
 
-            GeneratedTokenInfo generatedTokenInfo = jwtTokenBuilder.Build();
+            return jwtTokenBuilder.Build();
+        }
+
+        private RefreshToken GenerateRefreshToken(Guid generatedTokenId)
+        {
+            return new RefreshToken()
+            {
+                Id = Guid.NewGuid(),
+                JwtId = generatedTokenId,
+                ExpireDate = DateTime.UtcNow.AddMonths(_refreshTokenOptions.ExpiresOnMonth),
+                IsValidate = true
+            };
+        }
+
+        private async Task<AuthUserResultDto> RegisterUser(User user)
+        {
+            if (await _unitOfWork.Users.IsEmailExist(user.Email))
```

Comment:
https://docs.microsoft.com/ru-ru/dotnet/csharp/fundamentals/coding-style/coding-conventions#layout-conventions

Reply:
It will be better to get rid of the '!' symbol in the exceptions since exception thrown is already something bad :)

Reply:
Done.